metabase/metabase
# Drop full screen mode text size modifications

Closes https://github.com/metabase/metabase/issues/9615

### Description

Drop full screen mode font size modifications

### How to verify

1. go to dashboard, create text card, put some text that fills almost all space of the card
2. go to full screen mode (... -> enter full screen mode)
3. verify that font size in dashcards is not changed  

### Demo

https://www.loom.com/share/b8b8661e5cd44361999c89eee89b5f71

### Checklist

~- [ ] Tests have been added/updated to cover changes in this PR~
